src/client/mod.rs

```diff
@@ -1,14 +1,17 @@
 mod event_loop;
 mod pool_connection;
 
-use crate::client::event_loop::{ClientLoopCommand, ClientLoopWorker};
 use crate::config::ClientConfig;
 use crate::error::Error;
 use crate::irc;
 use crate::login::LoginCredentials;
 use crate::message::commands::ServerMessage;
 use crate::message::IRCMessage;
 use crate::transport::Transport;
+use crate::{
+    client::event_loop::{ClientLoopCommand, ClientLoopWorker},
+    message::{IRCTags, PrivmsgMessage},
+};
```

Comment:
Could you make these more in line with how the rest of the imports are done, aka usually single-line imports?

Reply:
This was made multi-line by rustfmt, the test fails otherwise.

Reply:
I meant like, split it up into several `use` statements like the other use lines